app/client/src/components/pages/Community/components/routes/CommunityTabs.js

```diff
@@ -90,7 +156,7 @@ const StyledTabs = styled(Tabs)`
       &[data-selected],
       &:hover,
       &:focus {
-        z-index: 1;
+        z-index: 1
```

Comment:
The semicolon got removed, which is taking away the background color for the selected tab. Can you add that back in?

```suggestion
        z-index: 1;
```

Reply:
Thank you for catching this!